test/lazy-imports_test.html

```diff
@@ -0,0 +1,51 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+  <head>
+    <title>lazy-imports test</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../web-component-tester/browser.js"></script>
+
+    <link rel="import" href="upgrade-button.html">
+  </head>
+  <body>
+
+    <test-fixture id="lazyImports">
+      <template>
+        <upgrade-button></upgrade-button>
+      </template>
+    </test-fixture>
+
+    <script>
+      var initTest = function() {
+        var element = fixture('lazyImports');
+        var button = element.$.button;
+        var lazy = element.$.lazy;
+        return {element: element, button: button, lazy: lazy};
+      }
+      suite('lazy-imports', function() {
+
+        test('After import triggered, element upgrades', function(done) {
+          var testEnvironment = initTest();
+          assert.isUndefined(testEnvironment.lazy.importLoaded);
+          testEnvironment.element.addEventListener("import-loaded", function(){
+            console.log(testEnvironment.lazy);
+            assert.isTrue(testEnvironment.lazy.importLoaded);
```

Comment:
Before seeing these comments, I changed it to a `true` property on the prototype. I didn't want this test to be locked to polymer 1.0 style property configuration.